project_resstock_multifamily/housing_characteristics/Geometry Number Units.tsv

```diff
@@ -1,7 +1,6 @@
+Mobile Home	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	1.0	541.0	6940961.26893
+Multi-Family with 2 - 4 Units	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	1.0	926.0	9013016.88311
+Multi-Family with 5+ Units	0.0386290341761	0.104259205091	0.0178355502537	0.163626334478	0.0108480518558	0.0554976731323	0.00564948847776	0.111498670185	0.00857090006186	0.0107938236438	0.0154978320143	0.0361782227475	0.00138967505045	0.0129170027487	0.000949786227711	0.0320911145575	0.00473694920449	0.0029208355188	0.00225759231417	0.0357075568591	0.0100569672977	0.00223819985144	0.00335024397948	0.0042399487653	0.0293550139931	0.000647179681013	0.00566684727808	0.00390787485804	0.00273696610507	0.00322292212511	0.0132664272126	0.00134073160475	0.000817433627095	0.0166623232963	0.000316582626164	0.00487737481116	0.000333592014533	0.00129016803374	0.00201251598368	0.00035468136046	0.000265200907281	0.00785566900795	0.00212110156852	0.0227246854263	0.000899829838604	0.000843042802986	0.000628754362388	0.00239173378369	0.000444831883454	0.00181948783541	0.00103734561786	0.00173389673662	0.00129435936203	0.0199439606601	0.00128632930977	0.00217739840279	0.00254426058758	0.00181418067784	0.00130196563475	0.00603905874126	0.00266696173927	0.00311056617373	0.00166149225951	0.00722397694031	0.000890917154985	0.000913387401467	0.00162494529195	0.00232218568716	0.0065837515547	0.000328288043617	0.00437775801412	0.000654749851286	0.0142191900816	0.000413006071897	0.000477790060298	0.000478313512099	0.00158466280946	0.00113550710793	0.000892939162208	0.000433784979223	0.000440428059716	0.0116092312065	0.00162167009545	0.001207054687	0.00167670150591	0.00149389646152	0.000425046819396	0.00289520872526	0.00324046212463	0.000615901621013	0.00117437322054	0.00317738339988	0.000397617676938	0.00193166918282	0.00116271400319	0.000710532151104	0.000721727767049	0.000478313534182	0.00144972586983	0.00120770948906	0.000544796517276	0.000344503404328	0.0113079199072	0.013394772208	0.0145582628919	0.00672457118339	0.00577724613969	0.0	1923.0	19109698.2999
```

Comment:
@mpathak24 @joseph-robertson @ejhw there are some super tiny percentages for some of these, seems like they should be binned more.

Reply:
Yes. Also what happened to Option=1, 2, etc.?